e2e-tests/src/test/scala/pl/touk/nussknacker/DockerBasedInstallationExampleNuEnvironment.scala

```diff
@@ -19,7 +19,7 @@ import java.io.{File => JFile}
 trait DockerBasedInstallationExampleNuEnvironment extends BeforeAndAfterAll with BeforeAndAfterEach with LazyLogging {
   this: Suite =>
 
-  private lazy val specSetupService = unsafeContainerByServiceName("spec-setup")
+  private val specSetupService = unsafeContainerByServiceName("spec-setup")
```

Comment:
The companion object of the trait `DockerBasedInstallationExampleNuEnvironment` is not instantiated automatically when the trait is instantiated - it is instantiated when it is first accessed. Without something like this non-lazy val refering to that companion object, the docker env is not started before test that extend this test suite.

A good answer on SO for this with Scastie worksheet is here:
https://stackoverflow.com/questions/55477793/order-of-initialization-in-scala